microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -91,3 +91,39 @@ groups:
         title: "Metadata Overhead Above 90% on Bucket: {{ $labels.bucket }}, Node: {{ $labels.instance }}"
         description: The percentage of memory that is taken up by metadata is over 90%
         remediation: Increase memory allocation for bucket or change the evictionPolicy of the bucket from `Value-only` (be aware this will have an adverse effect on performance).
+
+    - alert: CB90061-diskWriteQueueLength-Alert
+      expr: |
+        kv_ep_storage_age_seconds > 60
```

Comment:
I think it may be worth raising this threshold, given that `critical` translates to "should be woken up at night". What do you reckon?

Reply:
If it's to wake people up then yeah it should be higher. I've seen it hit 80 on clusters so would raising the warning to 50 and then alert to 100 be appropriate?

Reply:
I'd say alert should be at least 90, 100 may be a good threshold to put it there. 50 for warning may be good as well, or perhaps 60 and 120 for consistency? Up to you :)

Reply:
I've set it so the alert is at 100 and the warning is at 50, personally i feel like 120 is just a little too long.